packages/dds/tree/src/core/tree/visitDelta.ts

```diff
@@ -70,8 +74,18 @@ export function visitDelta(
 	const attachPassRoots: Map<ForestRootId, Delta.FieldMap> = new Map();
 	const rootTransfers: Delta.DetachedNodeRename[] = [];
 	const rootDestructions: Delta.DetachedNodeDestruction[] = [];
+	const refresherData: [Major, Minor, ITreeCursorSynchronous][] = [];
+	delta.refreshers?.forEach(({ id: { major, minor }, trees }) => {
+		for (let i = 0; i < trees.length; i += 1) {
+			const offsettedId = minor + i;
+			refresherData.push([major, offsettedId, trees[i]]);
+		}
+	});
```

Comment:
Is there any value to putting all the refreshers in a flat list first? It seems like it should be faster (and less code) to just call `setInNestedMap` within the for loop.